### ISSUE TITLE
Description for "Get time bound responses" needs cleanup

### ISSUE THREAD

**Issue by mwaterous** [opsramp/openapi-specs#56]
This is another one that still contains the query variables in the description:

https://github.com/opsramp/openapi-specs/blob/a5cb1095590f34d70add7dcb0bb449235d0253aa/v2/opsramp-tickets.v2.yaml#L14564

As well as these notes:

https://github.com/opsramp/openapi-specs/blob/a5cb1095590f34d70add7dcb0bb449235d0253aa/v2/opsramp-tickets.v2.yaml#L14577

I've seen these notes on a few endpoints. It seems like generic need to know information that should be contained somewhere in the getting started guide once, instead of repeated in multiple locations as endpoint descriptions. If updates need to be made to these notes, every instance has to be tracked down or each one could end up being unique and possibly out of sync with the rest.

**Comment by rshera01** [comment]
Yes, this needs to be generic and needs to be defined as introduction.

**Comment by mwaterous** [comment]
Maintaining this in multiple locations is going to lead to inconsistencies and other problems. Since we can't define this kind of information as a re-usable schema component in OpenAPI it would make sense that this should be included as part of a reference, either on the Getting Started page, or another like it. I'll bring it up during todays meeting and let's discuss any other possible solutions.
